fcm_django/models.py

```diff
@@ -318,7 +318,7 @@ def send_message(
             )
         except FirebaseError as e:
             self.deactivate_devices_with_error_result(self.registration_id, e)
-            return e
+            raise e
```

Comment:
can skip referring to exception variable, just `raise` will do

Reply:
@henriklindgren , so I'll change it simple to raise